[german_enhanced] Initial submission of German enhanced keyboard

xavernitsch opened the pull request:
Hello,
this is an enhanced version of [German Basic 1.1](https://keyman.com/keyboards/basic_kbdgr).

German Enhanced. The keyboard layout follows the Windows 10 layout. It is optimized to be used with macOS and an Apple keyboard with german layout.

Press `^`, `´` or `` ` `` twice to get the character.

I attached a compiled version [german_enhanced.zip](https://github.com/user-attachments/files/22680331/german_enhanced.zip)

### Build log
```text
$ rm -r -f build && kmc build
xavernitsch keyman_keyboards german_mac release-g_german_enhanced.kpj - info KM05002: Building xavernitsch keyman_keyboards german_mac release-g_german_enhanced.kpj
german_enhanced.kmn - info KM05002: Building source/german_enhanced.kmn
german_enhanced.kmn - info KM05006: source/german_enhanced.kmn built successfully.
german_enhanced.kps - info KM05002: Building source/german_enhanced.kps
german_enhanced.kps - info KM05006: source/german_enhanced.kps built successfully.
xavernitsch keyman_keyboards german_mac release-g_german_enhanced.kpj - info KM05002: Building xavernitsch keyman_keyboards german_mac release-g_german_enhanced.kpj
xavernitsch keyman_keyboards german_mac release-g_german_enhanced.kpj - info KM05006: xavernitsch keyman_keyboards german_mac release-g_german_enhanced.kpj built successfully.
xavernitsch keyman_keyboards german_mac release-g_german_enhanced.kpj - info KM0500B: Project xavernitsch keyman_keyboards german_mac release-g_german_enhanced.kpj built successfully.
```

keyman-server commented:
Thank you for your pull request. You'll see a "build failed" message until the Keyman team has reviewed the pull request and manually initiated the build process.

Every change committed to this branch will become part of this pull request. When you have finished submitting files and are ready for the Keyman team to review this pull request, please post a "Ready for review" comment.

xavernitsch commented:
Hi, I don't have access to the TeamCity logs. Can you attach them?

LornaSIL commented:
PS: we don't do a build until we've thoroughly checked keyboards, so it just automatically fails for new submissions.

xavernitsch commented:
hi @LornaSIL,
thank you for your review.

I changed copyright and added the symbols to the touch layout.

[build.zip](https://github.com/user-attachments/files/22709640/build.zip)

```
rm -rf build && npx kmc build
xavernitsch keyman_keyboards german_enhanced release-g_german_enhanced.kpj - info KM05002: Building xavernitsch keyman_keyboards german_enhanced release-g_german_enhanced.kpj
german_enhanced.kmn - info KM05002: Building source/german_enhanced.kmn
german_enhanced.kmn - info KM05006: source/german_enhanced.kmn built successfully.
german_enhanced.kps - info KM05002: Building source/german_enhanced.kps
german_enhanced.kps - info KM05006: source/german_enhanced.kps built successfully.
xavernitsch keyman_keyboards german_enhanced release-g_german_enhanced.kpj - info KM05002: Building xavernitsch keyman_keyboards german_enhanced release-g_german_enhanced.kpj
xavernitsch keyman_keyboards german_enhanced release-g_german_enhanced.kpj - info KM05006: xavernitsch keyman_keyboards german_enhanced release-g_german_enhanced.kpj built successfully.
xavernitsch keyman_keyboards german_enhanced release-g_german_enhanced.kpj - info KM0500B: Project xavernitsch keyman_keyboards german_enhanced release-g_german_enhanced.kpj built successfully.
```

LornaSIL commented:
I don't see that this was done. Where did you put it?
>Your touch layout does not seem to have added U+1E9E and possibly other changes you made. You probably at least want to add that. You don't have to add another layer to the touch layout. You could even add it as a longpress to U+00DF on the default layer.
